document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/data/accounts/AccountsManager.java

```diff
@@ -1,44 +1,70 @@
 package uk.gov.companieshouse.document.generator.accounts.data.accounts;
 
+import static uk.gov.companieshouse.document.generator.accounts.AccountsDocumentInfoServiceImpl.MODULE_NAME_SPACE;
+
+import java.util.HashMap;
+import java.util.Map;
 import org.springframework.http.HttpEntity;
 import org.springframework.http.HttpHeaders;
 import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
+import org.springframework.stereotype.Component;
 import org.springframework.web.client.RestTemplate;
 import uk.gov.companieshouse.api.model.accounts.Accounts;
 import uk.gov.companieshouse.environment.EnvironmentReader;
 import uk.gov.companieshouse.environment.impl.EnvironmentReaderImpl;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
 
 /**
  * Temporary solution until private-sdk has been completed (SFA-518, SFA-670). When completed, this
  * file will get removed alongside the data package and all references to this file will be replaced
  * with calls to the private-sdk.
  */
+@Component
 public class AccountsManager {
 
     /** represents the Authorization header name in the request */
     private static final String AUTHORIZATION_HEADER = "Authorization";
 
     private static final EnvironmentReader READER = new EnvironmentReaderImpl();
 
-    private static String API_URL = READER.getMandatoryString("API_URL");
-    private static String CHS_API_KEY = READER.getMandatoryString("CHS_API_KEY");
+    private final String API_URL = READER.getMandatoryString("API_URL");
+    private final String CHS_API_KEY = READER.getMandatoryString("CHS_API_KEY");
```

Comment:
If this is final it can only be instantiated once per instance created. Why don't you make it static so it only needs to get instantiated once? We should be loading the config when the service starts to fail the start-up if config is missing.

Reply:
Also, all caps snake case is the convention for static final variables. It should be camel case for final instance variables. While being a minor nitpick, it does add smell to sonar.

Reply:
I tried to make it static but the test fail as `Mockito` falls over. Also @jterrych  good spot, my IDE highlighted it but must have forgot